webapp/src/DOMJudgeBundle/Entity/ScoreCache.php

```diff
@@ -65,7 +65,12 @@ class ScoreCache
      */
     private $solvetime_public;
 
-
+    /**
+     * @var boolean
+     *
+     * @ORM\Column(type="boolean", name="is_first_to_solve", options={"comment"="Is this team the first to solve for this problem,sortorder?"}, nullable=false)
+     */
+    private $is_first_to_solve = false;
```

Comment:
why do we have to set this to `false` explicitly?

Reply:
Seems a reasonable default; something is only first when it has been explicitly been confirmed to be that.